components/builder-api-proxy/config/nginx.conf

```diff
@@ -70,11 +74,21 @@ http {
       rewrite ^(.*)$ https://$host$1 permanent;
     }
 
+    access_log {{pkg.svc_path}}/logs/host.access.log nginx;
```

Comment:
Will this rotate appropriately? What we really want to do is ensure that logs from journald get into a log aggregator like Sumo Logic (unless data dog also does that)

Reply:
Or maybe since it's the access log we really don't care for it in an aggregator?

Reply:
The Datadog agent will be able to parse it and we'll get the stats that way. The rotation is something I will figure out shortly - this is likely the first of a few tweaks on this.

Reply:
Yeah, we'll want to be careful about rotation or max file size. We've already learned that high traffic directly affects syslog file sizes which isn't surprising.

Reply:
Yuck, looks like nginx doesn't handle this itself: https://www.nginx.com/resources/wiki/start/topics/examples/logrotation/

Reply:
Yeah, I'm leaning toward using logrotate - it looks fairly straightforward to wire up.

Reply:
The other option is to rely solely on your aggregator. I did this on my last large project and it worked swimmingly

Reply:
Yeah the main reason atm to log to file with nginx is to get the response times